src/plugins/orangehrmMaintenancePlugin/Service/PurgeEmployeeService.php

```diff
@@ -75,6 +76,34 @@ public function purgeEmployeeData(int $empNumber): void
         }
     }
 
+    /**
+     * @param int $vacancyNumber
+     * @throws TransactionException
+     */
+    public function purgeCandidateData(int $vacancyNumber): void
```

Comment:
this function is a duplicate of `purgeEmployeeData` function. only different is the file name `self::GDPR_PURGE_CANDIDATE`. This is how it was done in 4.x. Shall I combine the two functions and have an argument for the file name?